content/_announcements/jan2020-new-website.md

```diff
@@ -4,25 +4,37 @@ title: Our new website
 announcement-date: January 20, 2020
 author: clarence
 toc: true
-tags: 
+tags:
  - news
  - website
 ---
 
-We have a new website. Hooray! Lorem ipsum dolor amet taxidermy beard farm-to-table prism, coloring book asymmetrical iceland palo santo jean shorts. Small batch sartorial skateboard salvia. Bushwick stumptown polaroid copper mug. Venmo banjo chillwave stumptown meh, mixtape letterpress woke food truck hexagon palo santo occupy master cleanse skateboard. Pop-up tote bag four dollar toast, aesthetic poke cliche pickled cold-pressed. Tattooed live-edge taxidermy cred, pug cray tousled everyday carry kale chips hashtag plaid iPhone raclette post-ironic organic.
 
-## Heading 
+# We have a new website.
```

Comment:
So if you use this as the title, it'll get rendered in the page layout in the top header thing. Otherwise we'll have two h1s right next to each other?

Reply:
I don't know what you mean by the top header thing. 

Should my headings all start at H2? I changed this one. 

I'll take a look at the Jekyll documentation. 
:- )